### PR TITLE
[FR-187] Implement graphql to api sticker data conversion

### PR DESCRIPTION
**Pull request by JohnyTheCarrot**

### CODE REVIEW

**Review by Yomanz — APPROVED** [review]
LGTM, but Advaith might have some concerns.

File: src/util/convertToDiscord/convertField.ts [hunk 1]
```diff
@@ -28,6 +33,19 @@ import { getIdFromUrl } from '@util/convertToDiscord/getIdFromUrl';
...
+      return StickerFormatType.PNG;
+    case FormatType.Apng:
+      return StickerFormatType.APNG;
+    case FormatType.Lottie:
+      return StickerFormatType.Lottie;
+    default:
+      return StickerFormatType.PNG;
+  }
+}
+
```
Comment (advaith1) [inline src/util/convertToDiscord/convertField.ts:48]: can we implement this with an object instead? are there benefits to using the switch?

**Review by advaith1 — APPROVED** [review]

File: src/util/convertToDiscord/convertField.ts [hunk 2]
```diff
@@ -209,6 +227,16 @@ export const convertField = {
...
 
+  stickers: (stickers: Sticker[]): APISticker[] =>
+    stickers.map(sticker => ({
+      id: sticker.id,
+      name: sticker.name,
+      format_type: stickerGqlFormatToAPIFormat(sticker.formatType),
+      description: null,
+      tags: '',
+      type: StickerType.Standard
+    })),
```
Comment (advaith1) [inline src/util/convertToDiscord/convertField.ts:238]: this should return [APIStickerItem](https://discord.com/developers/docs/resources/sticker#sticker-item-object)[] - looks like that's already what [renderer](https://github.com/widgetbot-io/message-renderer/blob/master/src/Content/Sticker/index.tsx) wants
just change the type and remove the unused fields

**Review by advaith1 — CHANGES_REQUESTED** [review]

**Comment by JohnyTheCarrot** [comment]
squash and merge to avoid the second commit

**Review by advaith1 — APPROVED** [review]

**Pull request merged**